lua/autorun/server/sv_daily_restart.lua

```diff
@@ -141,6 +293,10 @@ local function allRestartAlertsGiven()
     return table.Count( alertIntervalsInSeconds ) == 0
 end
 
+local function timeSinceStart()
+    return os.time() - SERVER_START_TIME
```

Comment:
What about `SysTime()` instead?

Reply:
I'd need to test whether or not SysTime() resets on changelevel.

Reply:
Ah right, it doesn't